test/Xpring/xpring-client.test.ts

```diff
@@ -8,7 +8,7 @@ import TransactionStatus from '../../src/XRP/transaction-status'
 import XpringClient from '../../src/Xpring/xpring-client'
 
 import XpringError from '../../src/Xpring/xpring-error'
-import TransactionResult from '../../src/XRP/model/transaction-result'
+import TransactionResult from '../../src/XRP/model/final-transaction-result'
```

Comment:
`FinalTransactionResult`?

Reply:
moved everything back to `TransactionResult` anyway